src/altscore/borrower_central/model/category.py

```
@@ -0,0 +1,372 @@
+class CategoryAsyncModule:
+    def __init__(self, altscore_client):
+        self.altscore_client = altscore_client
+
+    def renew_token(self):
+        self.altscore_client.renew_token()
+
+    def build_headers(self):
+        return build_headers(self)
+
+    @retry_on_401_async
+    async def create(self, new_category: dict, values: List[NewCategoryValueDTO] = []):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)